tools/docker/daml-all-in-one/supervisord.conf

```diff
@@ -20,6 +29,7 @@ autostart=true
 autorestart=true
 stderr_logfile=/var/log/daml.err.log
 stdout_logfile=/var/log/daml.out.log
+priority=30
```

Comment:
@raynatopedrajeta Please explain in a comment above this line why the priority 30 is needed

Reply:
Hello @petermetz ,

I set it to 30 since it will be the last line to be executed. Please let me know if adjustments will be made on the numbers of priority.

Reply:
@raynatopedrajeta Unless there is a specific reason for it (e.g. it is broken without it) I would just remove it so that the default priority takes effect.

Reply:
Hello @petermetz ,

I've included priority for the run to chronologically executes these DAML commands. I've included priority number to ensure that queuing of command execution. 10 is the first while 30 is the last.

Thank you!

Rayn